packages/web-embed-api/src/.npmignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything in the root
```

Comment:
I think root `.npmignore` already takes care of this.

Reply:
Yeah, this should have been removed. Will remove.